### PR TITLE
Update auto-update config of Vega-Lite

### PR DESCRIPTION
**Pull request by domoritz**
https://github.com/vega/vega-lite/pull/1957

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@domoritz Please use a meaningful PR title with the library name, thanks.

**Comment by domoritz** [comment]
> Should we add this? It seems like we should modify the old one?

I'd like to stay backwards compatible. We may re-release a 1.x version. 

> Also, why .css and .map?

For the future as suggested by @PeterDaveHello. Vega uses the same format.

File: ajax/libs/vega-lite/package.json [hunk 1]
```diff
@@ -16,6 +16,12 @@
       "files": [
         "vega-lite*.+(js|css|map)"
       ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "**/vega-lite*.+(js|css|map)"
```
Comment (pvnr0082t) [inline ajax/libs/vega-lite/package.json]: @domoritz I'd like to make sure If there are only distributed purpose files under `build` directory. If so, we can use `"**/*"` instead. If not, let's just keep this.

Comment (domoritz) [inline ajax/libs/vega-lite/package.json]: We build every single `.ts` file and put it in `build`. I don't think they should be on a cdn.

Comment (pvnr0082t) [inline ajax/libs/vega-lite/package.json]: @domoritz Okay, let's keep it. :+1:

**Comment by domoritz** [comment]
@PeterDaveHello Let me know if you want any additional changes.

**Pull request merged**